### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Fbouchezpro**

### CODE REVIEW

**Review by Fostrankill — DISMISSED** [review]

File: frontend/src/components/Gallery/Gallery.jsx [hunk 1]
```diff
@@ -1,21 +1,35 @@
-import Card from "@components/Cards/Card/Index";
+import { useState, useEffect } from "react";
+import axios from "axios";
+/* eslint-disable import/no-unresolved */
+/* eslint-enable import/no-unresolved */
```
Comment (LoicBrassart) [inline frontend/src/components/Gallery/Gallery.jsx]: ... Why disabling a rule and reenabling it just afterwards ? This shouldn't fix anything O_O

File: frontend/src/pages/Home.jsx [hunk 2]
```diff
@@ -1,3 +1,26 @@
+// eslint-disable-next-line import/no-unresolved
+import Carousel from "@components/Carousel/Carousel";
+// eslint-disable-next-line import/no-unresolved
+import Logo from "@components/Header/Logo/Logo";
+// eslint-disable-next-line import/no-unresolved
+import Titre from "@components/Header/Titre/Titre";
+// eslint-disable-next-line import/no-unresolved
+import SearchButton from "@components/SearchButton";
```
Comment (LoicBrassart) [inline frontend/src/pages/Home.jsx]: Replace with 
```
// eslint-disable import/no-unresolved
import Carousel from "@components/Carousel/Carousel";
import Logo from "@components/Header/Logo/Logo";
import Titre from "@components/Header/Titre/Titre";
import SearchButton from "@components/SearchButton";
// eslint-enable import/no-unresolved
```

File: frontend/src/components/Gallery/Gallery.jsx (hunk 1 above)
Comment (LoicBrassart) [reply frontend/src/components/Gallery/Gallery.jsx]: Nah, delete these two lines and recommit

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- Stop disabling rules whenever you feel like it :roll_eyes: 
- If you **have** to disable a rule, do it properly (and with my accord <3 )

File: frontend/src/pages/Home.jsx (hunk 2 above)
Comment (Fbouchezpro) [reply frontend/src/pages/Home.jsx]: Done sire !

File: frontend/src/components/Gallery/Gallery.jsx (hunk 1 above)
Comment (Fbouchezpro) [reply frontend/src/components/Gallery/Gallery.jsx]: Plz be kind Chief Tribo

**Review by LoicBrassart — APPROVED** [review]
Better, no more Evil

**Review by Fostrankill — APPROVED** [review]

**Pull request merged**